utils/userElegible.ts

```diff
@@ -0,0 +1,21 @@
+import { FiscalCode } from "@pagopa/ts-commons/lib/strings";
+import { FeatureFlag, getIsUserEligibleForNewFeature } from "./featureFlags";
+import { BETA_TESTERS, getConfigOrThrow } from "./config";
+
+const config = getConfigOrThrow();

```

Comment:
The only file that should contain `getConfigOrThrow` call is `index.ts`; never add it in other files, because when adding unit tests it became harmful.

So just prepare the `getIsUserElegibleIoWebProfile` and call it where we have the config available, that is the `handler.ts` file.